src/CONST.js

```diff
@@ -425,7 +425,11 @@ const CONST = {
         LARGE: 'large',
         DEFAULT: 'default',
     },
+
+    MAX_AVATARS: 4,
```

Comment:
Let's move this to a more relevant Area to `CONST.REPORT` and rename it to `MAX_PREVIEW_AVATARS`.

Reply:
ok done with it.